docs/pt/docs/tutorial/path-operation-configuration.md

```diff
@@ -0,0 +1,180 @@
+# Configuração da Operação de Rota
+
+Existem vários parâmetros que você pode passar para o seu *decorador de operação de rota* para configurá-lo.
+
+!!! warning "Aviso"
+    Observe que esses parâmetros são passados diretamente para o *decorador de operação de rota*, não para a sua *função de operação de rota*.
+
+## Código de Status da Resposta
+
+Você pode definir o `status_code` (HTTP) para ser usado na resposta da sua *operação de rota*.
+
+Você pode passar diretamente o código `int`, como `404`.
+
+Mas se você não se lembrar do que cada código numérico significa, pode usar as constantes de atalho em `status`:
```

Comment:
What do you think of `o que` instead of `do que` here?

```suggestion
Mas se você não se lembrar o que cada código numérico significa, pode usar as constantes de atalho em `status`:
```

Reply:
Thks!